pkg/proc/target_exec.go

```diff
@@ -107,6 +107,7 @@ func (grp *TargetGroup) Continue() error {
 			// might very well not work.
 			_ = grp.setCurrentThreads(traptgt, trapthread)
 			if pe, ok := contOnceErr.(ErrProcessExited); ok {
+				traptgt.ClearSteppingBreakpoints()
```

Comment:
I think this deserves a comment.

Reply:
Will do.

(I was actually a little worried that trying to clear breakpoints on a process that had exited (when using the standard, non-replay backend) might break things here but it appears to be fine, so just want a comment to make sure it's clear who wanted it!)

Reply:
I thought this issue would probably affect rr too but so far I've not been able to repro with that backend.  There must be some subtlety involved...  Still, I think this is a reasonable change overall so I'm going to stick with it anyhow.